src/Sleep.js

```diff
@@ -17,40 +17,22 @@ class Sleep {
     return [...specificUser].splice(-7 * week, 7).map(day => day.date);
   }
 
-  returnAvgSleepHours() {
+  returnAvgSleepInfo(metric) {
     let specificUser = this.findUser();
-    return Number((specificUser.reduce((totalHours, day) => {
-      totalHours += day.hoursSlept;
-      return totalHours;
+    return Number((specificUser.reduce((totalMetric, day) => {
+      totalMetric += day[metric];
+      return totalMetric;
     }, 0) / specificUser.length).toFixed(2));
   }
 
-  returnAvgSleepQuality() {
+  returnSleepInfo(date, metric) {
     let specificUser = this.findUser();
```

Comment:
@NaomiKC92  What do you think about naming this method returnDaysSleepInfo so it starts parallel with the naming convention you used in Activity?

Reply:
Sounds good, just took care of it